lib/resources/utils.dart

```diff
@@ -23,6 +23,7 @@ class Utils {
     AuthProvider authProvider =
         Provider.of<AuthProvider>(context, listen: false);
     authProvider.signOut(context);
-    Navigator.pushReplacementNamed(context, Routes.login);
+    Navigator.popUntil(context, (route) => !route.isFirst);
+    Navigator.pushNamed(context, Routes.login);
```

Comment:
Use `pushNamedAndRemoveUntil` with `(route) => false`. It should work fine (remember to test it though).

Reply:
It works